### DISCUSSION TITLE
[New Feature] Allow debris animations not to damage walls

### DISCUSSION THREAD

**Discussion by Crimsonum** [Vinifera-Developers/Vinifera#1276] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

When a debris animation (animation type with `Bouncer=yes`) collides with a wall, the wall is damaged. This happens even if the animation's warhead should not allow damage to walls (`Wall=no`, which is the default for warheads). This creates a drawback/limitation for invisible debris anims, which are commonly used by TS modders as a workaround to deliver damage over area.

### Proposed Code (optional)

The vanilla game has only a few types of debris animations, metallic debris (`DBRIS-`), meteors (`MET-`), and the unused Tiberium shards (`CRYSTAL#`). Both the metallic debris and crystals use the `TankOGas` warhead, which has `Wall=yes` set. The meteors are a special case, they have no valid warhead specified. AFAIK this means the game uses the `C4Warhead= --> HE` warhead to deliver the damage from a meteor impact, which too has `Wall=yes` set.

In other words, removing this hardcoded feature altogether should benefit modders without affecting the vanilla game.
